services/movie.py

```diff
@@ -15,18 +20,22 @@ def get_movies(
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__contains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
```

Comment:
The `get()` method is used without handling the `DoesNotExist` exception. Consider using `get_object_or_404()` or wrapping this call in a try-except block to handle the exception and avoid runtime errors.